### PR TITLE
Service/list-routes

### PR DESCRIPTION
**Pull request by GonzaNunezCano**
### Brief Description
 The service is available to list all the existing routes of an organization 
### Related card or ticket
-- Add here a link to the related card or ticket --
https://github.com/Pis-moove-it/reciclando-backend/issues/35

### CODE REVIEW

File: public/docs/swagger.yaml [hunk 1]
```diff
@@ -516,6 +516,26 @@ paths:
...
+            $ref: '#/definitions/Error'
+      security:
+        - ApiKey: []
+    get:
+      tags:
+        - "Routes"
+      operationId: listingRoutes
+      summary: Listing all the routes from the organization.
       responses:
         200:
```
Comment (gastonabella) [inline public/docs/swagger.yaml:571]: Here you are not listing the routes in an array, as in  200 OK responses in other GET services

File: public/docs/swagger.yaml [hunk 2]
```diff
@@ -733,6 +753,50 @@ definitions:
...
+      id:
+        type: integer
+      length:
+        type: number
+      travel_image:
+        type: string
+      created_at:
+        type: string
+      user:
+        type: object
```
Comment (gastonabella) [inline public/docs/swagger.yaml]: Here you can replace `object` with `'#/definitions/User'`, and not defining the User again

**Review by gastonabella — CHANGES_REQUESTED** [review]

File: app/controllers/base_controller.rb [hunk 3]
```diff
@@ -15,4 +15,18 @@ def render_error(error_code, error_details = '')
     render json: Error.error_for(error_code, error_details),
            status: Error.status_for(error_code)
   end
+
+  def paginated_render(query, page, per_page)
+    count = query.count
+    response.headers['records'] = count
+    if page && per_page
+      count = query.count
```
Comment (fbalarini) [inline app/controllers/base_controller.rb]: Repeated code

File: app/controllers/base_controller.rb [hunk 4]
```diff
@@ -15,4 +15,18 @@ def render_error(error_code, error_details = '')
...
+      page_count = (count / per_page.to_f).ceil
```
Comment (fbalarini) [inline app/controllers/base_controller.rb]: What do you think about moving this to another method? I think this logic is making this method more difficult to understand. What's more, maybe we can set `total_pages` to be 1 as default and use mod `%` operator when `count > per_page`.

**Review by fbalarini — APPROVED** [review]

**Review by gastonabella — APPROVED** [review]

**Pull request merged**